pkg/datastore/test/watch.go

```diff
@@ -727,7 +727,7 @@ func WatchCheckpointsTest(t *testing.T, tester DatastoreTester) {
 	require.NoError(err)
 
 	changes, errchan := ds.Watch(ctx, lowestRevision, datastore.WatchOptions{
-		Content:            datastore.WatchCheckpoints | datastore.WatchRelationships | datastore.WatchSchema,
+		Content:            datastore.WatchCheckpoints | datastore.WatchRelationships | datastore.WatchSchema, //
```

Comment:
```suggestion
		Content:            datastore.WatchCheckpoints | datastore.WatchRelationships | datastore.WatchSchema,
```

Reply:
sorry - I marked all tests locally that could be affected by this to review them. fixed.